skeleton/default.nix

```diff
@@ -0,0 +1,60 @@
+{ system ? builtins.currentSystem
+, crossSystem ? null
+, config ? {}
+# Import IOHK common nix lib
+, iohkLib ? import ./nix/iohk-common.nix { inherit system crossSystem config; }
```

Comment:
I've seen this referred to as all of:
- `iohkLib`
- `iohkCommon`
- `iohkNix`

You've  got both the first two here ;) Can we pick one and stick to it?